src/k8s/requests/v2v/startV2VvmwareController.js

```diff
@@ -12,21 +23,42 @@ const { info } = console;
 
 const OLD_VERSION = 'OLD_VERSION';
 
+// prevent parallel execution of startV2VVMWareController()
+let semaphor = false;
+
 // The controller is namespace-scoped, especially due to security reasons
 // Let's make sure its started within the desired namespace (which is not by default).
 // The V2VVmware CRD is expected to be already created within the cluster (by Web UI installation)
+// TODO: The controller should be deployed by a provider and not via following UI code.
 export const startV2VVMWareController = async ({ namespace }, { k8sGet, k8sCreate, k8sKill, k8sPatch }) => {
   if (!namespace) {
     throw new Error('V2V VMWare: namespace must be selected');
   }
 
+  if (semaphor) {
```

Comment:
this does not take other namespaces and new versions into account

Reply:
Can this happen within the single tab?

Reply:
I guess the versions probably not, but the namespaces certainly can - user can just select another namespace

Reply:
Right, this can happen. There is not much time for it, but it's possible. Will update the PR for this case.

Reply:
done